test/unit/features/instance/methods-events.spec.js

```diff
@@ -56,6 +56,13 @@ describe('Instance methods events', () => {
     expect(spy).toHaveBeenCalledWith(1, 2, 3)
   })
 
+  it('$once', () => {
```

Comment:
Let's use a more specific description for this test case. eg. "$off event added by $once"

Reply:
thanks, and I had do this.

Reply:
```suggestion
  it('$off event added by $once', () => {
```